modules/solace/options.go

```diff
@@ -44,10 +38,11 @@ func (o Option) Customize(*testcontainers.GenericContainerRequest) error {
 	return nil
 }
 
-// WithExposedPorts allows adding extra exposed ports
-func WithExposedPorts(ports ...string) Option {
+// WithServices configures the services to be exposed with their wait strategies
+func WithServices(srv ...Service) Option {
 	return func(o *options) error {
-		o.exposedPorts = append(o.exposedPorts, ports...)
+		// Clear existing services and use only the specified ones
```

Comment:
bug: if the passed slice is empty, then the container will not expose any service. I'd return an error here for that case

Reply:
yeah, you are right, updated this part and covered with a test